ansible/roles/docker/tasks/rhel-centos.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Add Docker Repository
+  yum_repository:
+    name: docker-ce
+    baseurl: https://download.docker.com/linux/centos/docker-ce.repo
+
+- name: Install Docker & Dependencies
+  yum:
+    name: "{{ packages }}"
+  vars:
+    packages:
+      - yum-utils
+      - device-mapper-persistent-data
+      - lvm2
+      - docker-ce-18.09.6-3.el7
+      - docker-ce-cli-18.09.6-3.el7
+      - containerd.io
+
+- name: Enable and Start Docker
+  systemd:
+    state: started
+    name: docker
+    enabled: yes
```

Comment:
Yeah, but I usually try to avoid 3rd-party Ansible roles/playbooks because they often pack more tasks than what are required and quickly bloat the play. It can sometimes make it more difficult to debug. For something as small as the docker install, I think it's actually easier just to maintain a small Domino docker role. Though this could change in the future